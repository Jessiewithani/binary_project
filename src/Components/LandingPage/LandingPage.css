.landing-page {
    align-items: center;
    background-color: black;
    background-image: url('../../assets/giphy.gif') ;
    background-position: center;
    background-repeat: no-repeat;
    background-size: 50em;
    display:flex;
    height: 100vh;
    justify-content: center center;
}

.enter {
    color: white;
    font-family: 'get-schwifty';
    font-size: 4em;
    left: 5.3em;
    position: relative;
}

.link-to-enter {
    text-decoration: none;
}