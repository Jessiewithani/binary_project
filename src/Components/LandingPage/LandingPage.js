import React from 'react';
import '../LandingPage/LandingPage.css'
import { Link } from 'react-router-dom'

const LandingPage = () => {
    return(
        <div className="landing-page">
            <Link className="link-to-enter" to='/characters'><h1 className="enter">CLICK MY TERRY FOLD</h1></Link>
        </div>
    )
}

export default LandingPage;