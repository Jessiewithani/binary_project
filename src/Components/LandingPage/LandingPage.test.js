import React from 'react';
import LandingPage from '../LandingPage/LandingPage';
import { shallow } from 'enzyme'

describe('App', () => {
    it('should match the snapshot with the correct data passed in', () => {
      const wrapper = shallow(<LandingPage/>)
      
      expect(wrapper).toMatchSnapshot()
    });
    
})
  