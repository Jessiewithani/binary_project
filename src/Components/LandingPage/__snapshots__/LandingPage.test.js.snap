// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot with the correct data passed in 1`] = `ShallowWrapper {}`;
